railties/test/application/rake/notes_test.rb

```diff
@@ -126,6 +126,19 @@ def teardown
         end
       end
 
+      test 'notes finds notes in files with invalid encoding' do
+        from = File.join(framework_path, "railties/test/cases/encoding_iso8859_with_invalid_utf8.css")
+        to = File.join(app_path, "app/assets/stylesheets/encoding_iso8859_with_invalid_utf8.css")
+        FileUtils.mkdir_p File.dirname(to)
+        FileUtils.cp(from, to)
+
+        run_rake_notes do |output, lines|
+          assert_match(/handle notes in binary/, output)
```

Comment:
this passes in this PR, but fails when the `encoding:` arg in the diff is removed